helm_deploy/licences/Chart.yaml

```diff
@@ -12,5 +12,5 @@ dependencies:
     version: 0.4.1
     repository: https://ministryofjustice.github.io/hmpps-helm-charts
   - name: generic-data-analytics-extractor
-    version: 0.1.3
+    version: 0.1.4
```

Comment:
This now uses the python scripts. We're having some issues with the cronjob, so though updating to use the python scripts may help / be easier to get help with